lib/utils.js

```diff
@@ -505,23 +533,26 @@ function isPromiseLike(maybePromise) {
   return maybePromise && typeof maybePromise.then === 'function';
 }
 
-exports.filterOptions = filterOptions;
-exports.mergeOptions = mergeOptions;
-exports.translateOptions = translateOptions;
-exports.shallowClone = shallowClone;
-exports.getSingleProperty = getSingleProperty;
-exports.checkCollectionName = checkCollectionName;
-exports.toError = toError;
-exports.formattedOrderClause = formattedOrderClause;
-exports.parseIndexOptions = parseIndexOptions;
-exports.normalizeHintField = normalizeHintField;
-exports.handleCallback = handleCallback;
-exports.decorateCommand = decorateCommand;
-exports.isObject = isObject;
-exports.debugOptions = debugOptions;
-exports.MAX_JS_INT = 0x20000000000000;
-exports.mergeOptionsAndWriteConcern = mergeOptionsAndWriteConcern;
-exports.translateReadPreference = translateReadPreference;
-exports.executeOperation = executeOperation;
-exports.applyWriteConcern = applyWriteConcern;
-exports.isPromiseLike = isPromiseLike;
+module.exports = {
+  filterOptions,
+  mergeOptions,
+  translateOptions,
+  shallowClone,
+  getSingleProperty,
+  checkCollectionName,
+  toError,
+  formattedOrderClause,
+  parseIndexOptions,
+  normalizeHintField,
+  handleCallback,
+  decorateCommand,
+  isObject,
+  debugOptions,
+  MAX_JS_INT: 0x20000000000000,
```

Comment:
This is not related to the actual pull request, but fun fact, this is the same as `Number.MAX_SAFE_INTEGER + 1`. I feel like we should be using the built-in.